tests/test_models.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+test_models
+----------------------------------
+
+Tests for `teamsupport.models` module.
+"""
+import unittest
+
+from lxml import etree
+
+from teamsupport.errors import MissingArgumentError
+from teamsupport.models import Action, Ticket
+from tests import BaseTeamSupportServiceCase
+
+
+class TestTicket(BaseTeamSupportServiceCase):
+    def test_initialisation_with_id(self):
+        self.response.content = """<Ticket><TicketID>ID</TicketID></Ticket>"""
```

Comment:
Fixed in https://github.com/yola/teamsupport-python/commit/b55333c8f7563d85a9af367d32007c0b2bda6487